src/main/java/com/Project/accountManager/entities/User.java

```diff
@@ -0,0 +1,24 @@
+package com.Project.accountManager.entities;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Entity
+@Table(name="user")
+@Data
+public class User {
+    @Id
+    private Long id;
+    String name;
+    String surName;
+    int password;
+    public void setId(Long id) {
```

Comment:
Same like here ->https://github.com/serhatbsts/accountManager/pull/2/files#r1612713627

Reply:
OK, necessary corrections will be made.